diff --git a/lib/app/ui/pages/home/base/interface/serial_number_scan_interface.dart b/lib/app/ui/pages/home/base/interface/serial_number_scan_interface.dart
index bf370b7e8b259dcc55bf1d52a31b45478fdc411c..86134b4fa36f49bffa0a87f40de735367b1398be 100644
--- a/lib/app/ui/pages/home/base/interface/serial_number_scan_interface.dart
+++ b/lib/app/ui/pages/home/base/interface/serial_number_scan_interface.dart
@@ -1,6 +1,7 @@
 import 'package:basement/model.dart';
 import 'package:basement/repository.dart';
 import 'package:desktop/app/service/tcp_serial/tcp_socket_service/mixin/tcp_socket_getx_listener_mixin.dart';
+import 'package:desktop/app/ui/pages/home/base/base_form/base_form_controller.dart';
 import 'package:desktop/app/ui/pages/home/base/interface/submit_interface/submit_interface.dart';
 import 'package:desktop/app/ui/widget/num_pad/num_pad_util.dart';
 import 'package:desktop/app/utils/app_config.dart';
@@ -11,7 +12,8 @@ import 'package:get/get.dart';
 
 ///序列号不间断接收
 mixin SerialNumberScanInterface<T>
-on TcpSocketGetxListenerMixin<T>,
+on BaseFormController,
+    TcpSocketGetxListenerMixin<T>,
     SubmitInterface {
 
   ///序列号不间断接收列表
@@ -26,7 +28,7 @@ on TcpSocketGetxListenerMixin<T>,
     required String? invCCode,
   }) async {
     await Future.doWhile(() async {
-      if (_isLoading){
+      if (_isLoading || isLoading){ ///isLoading 等待报工结束
         await Future.delayed(const Duration(milliseconds: 100));
         return true;
       }
@@ -220,10 +222,14 @@ on TcpSocketGetxListenerMixin<T>,
         orderSNOnChanged([orderSNModel ?? MoOrderSNModel(id: string, code: string)]);
         if (autoCommitSubmit){
           ///此时所有报工数据都已填写完成，符合自动报工的条件，直接提交报工记录
-          _isLoading = false;
           update();
           serialNumberBarcodeMap.addAll({string: 200});
-          await saveSubmit(false, byAutoSubmit: true);
+          try {
+            await saveSubmit(false, byAutoSubmit: true);
+          }
+          finally {
+            _isLoading = false;
+          }
           return;
         }
         serialNumberBarcodeMap.addAll({string: 200});
@@ -259,10 +265,14 @@ on TcpSocketGetxListenerMixin<T>,
           return exit(msg: checkMap[false]!);
         }
         ///符合自动报工的条件，直接提交报工记录
-        _isLoading = false;
         update();
         serialNumberBarcodeMap.addAll({string: 200});
-        await saveSubmit(true, byAutoSubmit: true);
+        try {
+          await saveSubmit(true, byAutoSubmit: true);
+        }
+        finally {
+          _isLoading = false;
+        }
         return;
       }
       serialNumberBarcodeMap.addAll({string: 200});
@@ -279,4 +289,4 @@ on TcpSocketGetxListenerMixin<T>,
     required String? invCCode,
   }) get serialNumberScanOnBarcode => _serialNumberScanOnBarcode;
 
-}
\ No newline at end of file
+}
