diff --git a/lib/app/routes/mine_get_delegate.dart b/lib/app/routes/mine_get_delegate.dart
index 78d3da6b7319d97aeb0cedd162d772b78fae06e7..bb0cd53be9247c5467bdd3d1d3da7e178b1990f9 100644
--- a/lib/app/routes/mine_get_delegate.dart
+++ b/lib/app/routes/mine_get_delegate.dart
@@ -5,6 +5,7 @@ import 'package:basement/utils.dart';
 import 'package:desktop/app/routes/app_pages.dart';
 import 'package:desktop/app/routes/app_routes.dart';
 import 'package:desktop/app/service/tcp_serial/serial_com_service/mixin/serial_port_getx_listener_mixin.dart';
+import 'package:desktop/app/service/tcp_serial/tcp_socket_service/mixin/tcp_socket_getx_listener_mixin.dart';
 import 'package:desktop/app/ui/pages/home/base/base_form/base_form_controller.dart';
 import 'package:desktop/app/ui/pages/home/base/base_form/base_form_page.dart';
 import 'package:desktop/app/ui/pages/home/base/base_tab/base_tab_page.dart';
@@ -115,6 +116,11 @@ class MineGetDelegate extends GetDelegate {
               (pageWidget.controller as SerialPortGetXListenerMixin).enableSerialPort = true;
             }
 
+            if (pageWidget.controller is TcpSocketGetxListenerMixin){
+              writeLog('${pageWidget} enableTcpSocket true');
+              (pageWidget.controller as TcpSocketGetxListenerMixin).enableTcpSocket = true;
+            }
+
           } catch (e){}
         }
       }
@@ -167,6 +173,10 @@ class MineGetDelegate extends GetDelegate {
             writeLog('${pageWidget} enableSerialPort false beforePush');
             (pageWidget.controller as SerialPortGetXListenerMixin).enableSerialPort = false;
           }
+          if (pageWidget.controller is TcpSocketGetxListenerMixin){
+            writeLog('${pageWidget} enableTcpSocket false beforePush');
+            (pageWidget.controller as TcpSocketGetxListenerMixin).enableTcpSocket = false;
+          }
         } catch(e){}
       }
     }
@@ -209,6 +219,11 @@ class MineGetDelegate extends GetDelegate {
           controller.enableSerialPort = true;
         }
 
+        if (controller is TcpSocketGetxListenerMixin){
+          writeLog('${controller} enableTcpSocket true');
+          controller.enableTcpSocket = true;
+        }
+
         return false;
       } catch (e){
         return true;
diff --git a/lib/app/service/tcp_serial/tcp_socket_service/implement/tcp_socket.dart b/lib/app/service/tcp_serial/tcp_socket_service/implement/tcp_socket.dart
index edc7c753b1ddac57e837f0057656452f63caf7d5..8dd0b2205a8567e0d9bbc65c81e6fd53cf550cb7 100644
--- a/lib/app/service/tcp_serial/tcp_socket_service/implement/tcp_socket.dart
+++ b/lib/app/service/tcp_serial/tcp_socket_service/implement/tcp_socket.dart
@@ -35,9 +35,16 @@ class TcpSocket extends TcpSocketInterface {
       sock = await Socket.connect(host, port);
     } catch (e){
       isOpen = false;
+      errMsg = 'TCP $host:$port 连接时出错：${e.toString()}';
       WidgetsBinding.instance.addPostFrameCallback((timeStamp) async {
-        ToastNotification(Get.overlayContext!).error('TCP 连接时出错：${e.toString()}');
+        ToastNotification(Get.overlayContext!).error(errMsg);
       });
+      eventBus.fire(TcpSocketDataModel(
+        host: host,
+        port: port,
+        data: jsonEncode({'-1': errMsg}),
+        isConnectMsg: true,
+      ));
       return false;
     }
 
@@ -71,7 +78,10 @@ class TcpSocket extends TcpSocketInterface {
       }
     }, onError: (error) async {
       isOpen = false;
-      errMsg = 'TCP 连接出错：${error.toString()}，请检查！';
+      errMsg = 'TCP $host:$port 连接出错：${error.toString()}，请检查！';
+      WidgetsBinding.instance.addPostFrameCallback((timeStamp) async {
+        ToastNotification(Get.overlayContext!).error(errMsg);
+      });
       eventBus.fire(TcpSocketDataModel(
         host: host,
         port: port,
@@ -80,7 +90,10 @@ class TcpSocket extends TcpSocketInterface {
       ));
     }, onDone: () async {
       isOpen = false;
-      errMsg = 'TCP 连接关闭！';
+      errMsg = 'TCP $host:$port 连接关闭！';
+      WidgetsBinding.instance.addPostFrameCallback((timeStamp) async {
+        ToastNotification(Get.overlayContext!).error(errMsg);
+      });
       eventBus.fire(TcpSocketDataModel(
         host: host,
         port: port,
